src/Lykke.Service.HFT.Wamp/Auth/AnonymousSessionAuthenticator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using WampSharp.V2.Authentication;
+using WampSharp.V2.Core.Contracts;
+
+namespace Lykke.Service.HFT.Wamp.Auth
+{
+    public class AnonymousSessionAuthenticator : WampSessionAuthenticator
```

Comment:
As I understand it is not used and probably should be removed.

Reply:
Sure, I'll delete this class.